src/less/variables/ds6/border-radius-variables.less

```diff
@@ -0,0 +1,7 @@
+@border-radius-none: 0;
+@border-radius-form-control: 0;
+
+@border-radius-button: 20px;
+@border-radius-large-button: 24px;
+
+@border-radius-small: 8px;
```

Comment:
Sorry @agliga, I think you were closer first time around before my slack comment. `@border-radius-notice` and `border-radius-overlay`, would tie in better as a token scheme rather than using sizes (e.g. small, medium, large)

Reply:
Got it, reverted my change and used the notice and overlay.
I left buttons as is for now, might combine them as we round more components.